### DISCUSSION TITLE
Possible Internationalization approaches in WCAG 3

### DISCUSSION THREAD

**Discussion by rachaelbradley** [w3c/wcag3#17] [General]
WCAG 3 adds the concept of outcome with [conditional tests ](https://www.w3.org/TR/wcag-3.0/#conditions)(outcomes are the new version of success criteria).  One condition would be language used. 

This would likely apply to outcomes on things like line spacing, diacritics, and pronunciation. 

We would like an approach to handle internationalization in WCAG 3.  It is not possible with existing resources to test with every language worldwide. 

This Discussion thread is to start the conversation around possible approaches to internationalization in WCAG 3.

**Comment by rachaelbradley** [comment]
Initial suggested approach:

1. Agree on a set of languages that would be tested and have conditional outcomes written as part of the initial WCAG 3 document.
2. Create a process and guidance on how to evaluate whether an outcome applies and, if so, write the conditional test(s).
3. Apply that process when WCAG is translated to new languages.
4. Add the resulting new content to WCAG 3

**Comment by chaals** [comment]
Just checking that I understand the approach, an example could be

1. We decide to test in english, spanish, hindi, toki pona, and chinese and we write a conditional outcomes for english that says the impersonal construction (e.g. "it is known that" without saying who knows it) doesn't appear, but we don't apply that for spanish.
2. We explain that where a language commonly uses that construction, we expect people to understand it, but where it is generally only used in university-level writing it's likely not to be easy to understand so it should apply.
3. When someone translates WCAG 3 to japanese, they decide to apply the conditional outcome as per spanish, based on it being a common feature at primary-level (Note: I don't know if that's true, it's a hypothetical example I am inventing).
4. We add to WCAG 3 a statement that the outcome doesnt apply to japanese, as well as spanish.

Another example:

1. We have a conditional outcome that spoken audio must be available at a maximum speed of 100 words per minute for english, with other values for other languages we agreed to test.
2. We state that the maximum speed should be 2/3 of the normal speech rate for news presentations on "primary news radio" (again, I'm making this up, that's a pretty woolly description and I am sure there are better metrics available).
3. WCAG 3 is translated to bislama, and the translation team set the maximum speed for bislama at 120 wpm.
4. We add that conditional outcome WCAG 3.

*IF* I have understood the rough idea, it seems to make sense at first glance. I can see it leading us down a lot of lines that are not about internationalisation. I think working with it in the somewhat constrained context of internationalisation to see how it goes, and then assessing the approach and what to do with it, is a good way to deal with the idea.

I can imagine outcomes that say diacritics differentiating vowels must be present in arabic and hebrew, but diacritics are not required for english, and then (again as hypothetical examples) translators deciding that they are required for french but not spanish.

I can also imagine suggestions that where there are day/night styles provided, something doesn't have he same outcomes as where there is only a single default style, but I am not sure that internationalisation is the determinant there so we agree to postpone the discussion until we have tried it out some more. But that only seems to make sense if we gain experience with translations while we're drafting, which isn't the normal W3C approach to translation.

**Reply by rachaelbradley** [reply]
Slight adjustment from my perspective:
1. We select a set of languages to write for. Hypothetically: English, Hebrew, Spanish, Chinese and Arabic.  (Personally, I'd like to keep the initial set the same across the initial draft but that is another conversation)
2. We write a conditional outcome that when it applies, words should include all letters and diacritics needed to determine the meaning of the word. 
3. We write conditional tests for Hebrew and Arabic and note that this doesn't apply for English, Chinese, and Spanish.
4. We write a a procedure for translators to determine if the Conditional Outcome applies (In this case, determine if the language leave out vowels.) and writing the resulting conditional test or not applicable statement.

**Reply by johnfoliot** [reply]
> We select a set of languages to write for: English, Hebrew, Spanish, Chinese and Arabic. 

Has this list of languages been finalized, or is this just for illustration purposes? 

Respectfully, while Hebrew is indeed an important language to include, from a "biggest bang for your buck" (idiom: to optimize the deliverable) perspective I would love to have the WG revisit the stats I provided in [updates from Clear Language (outcomes only)](https://github.com/w3c/wcag3/pull/9#top) related to languages used globally. I have concerns that languages such as Hindi, French, Russian, Portuguese, and Japanese would be "left behind" 

- English – 1.452 billion speakers
- Mandarin Chinese – 1.118 billion speakers
- Hindi – 602 million speakers
- Spanish – 534 million speakers
- Arabic – 372 million speakers
- French – 300 million speakers
- Bengali – 265 million speakers
- Russian – 258 million speakers
- Portuguese – 223 million speakers
- Swahili (also known as Kiswahili) – 200 million speakers
- Indonesian – 200 million speakers
- Urdu – 170 million speakers
- Japanese – 150 million speakers
- German – 135 million speakers
- Punjabi – 125 million speakers
- Farsi (Persian) – 110 million speakers
- Javanese – 98 million speakers
- Italian – 85 million speakers
- Turkish – 85 million speakers

(source: https://lingua.edu/the-most-spoken-languages-in-the-world/)

**Reply by rachaelbradley** [reply]
The list of languages hasn't even been discussed. This thread was using hypotheticals to illustrate a proposed approach.  I've added clarification.
